frame/session/src/tests.rs

```diff
@@ -345,10 +345,9 @@ fn return_true_if_more_than_third_is_disabled() {
 		force_new_session();
 		initialize_block(2);
 
+		assert_eq!(Session::disable_index(0), true);
```

Comment:
Its not obvious to me why 0 would return true here. The way I read this 0 should not be a validator thus we would always expect false

Reply:
The API takes a validator index and not the validator id, it's just a bit confusing because in the test environment the validator id are `usize`. So in this case `0` refers to validator id `1` above.